### PR TITLE
adding the space adventure task to the library.

### PR DESCRIPTION
**Pull request by AlvaroAguilera**
We would like to contribute this probabilistic sequential decision-making task to the library.

### CODE REVIEW

**Comment by AlvaroAguilera** [comment]
was this the wrong repository to add new tests?

**Comment by vsoch** [comment]
Yes! Apologies I missed your initial post - we were running a large, cross cloud performance study at the end of August and I didn't look at anything else.

To answer your question - overall, yes, but if you want to contribute to expfactory.org you'll want to talk to @rwblair. For this repository, before we merge here you do need the preview url https://expfactory-experiments.github.io/space_adventure_pd to be actually live - do you want to add the experiment here? I can create a repository for you to PR to with the experiment files.

**Comment by AlvaroAguilera** [comment]
It would be great if you could add the repository since we don't have a freely available preview of the test. Thank you.

**Comment by vsoch** [comment]
All set! https://github.com/expfactory-experiments/space_adventure_pd. You can take a look at the other respositories to get a sense of the organization. In summary - most of the experiment files you are adding here should go there, and then only the metadata is added in this PR. Ping me in an issue there if you have questions. And please expect my responses to not be immediate - I'm on travel until early October.

**Comment by AlvaroAguilera** [comment]
I just uploaded the code to the new repo.

**Comment by vsoch** [comment]
Excellent! See if you can add some notes to the README, and then deploy the site to GitHub pages and put the URL in the description so a visitor can easily preview it.

**Comment by AlvaroAguilera** [comment]
Done. I think.

**Comment by vsoch** [comment]
It looks great! :space_invader: 

You should now be able to update the PR here to remove the experiment assets (they are in the external repository) and just add the yaml that describes that location.

**Comment by AlvaroAguilera** [comment]
I'm not sure what I should do now, Vanessa. Can you ELI5?

**Comment by vsoch** [comment]
You only need to add one file here: `docs/_library/space_adventure_pd.md` that should point to the repository where the experiment assets are, where preview is enabled, etc.

**Comment by AlvaroAguilera** [comment]
I think I removed everything as requested. Please merge. Thank you.

**Review by vsoch — APPROVED** [review]
Yes, you did a great job! Let's merge.
